content/api/commands/selectfile.md

```diff
@@ -0,0 +1,276 @@
+
+**<Icon name="angle-right"></Icon> file** **_(String, Array, Object or
+Buffer)_**
+
+Either a single file, or an array of files. A file can be:
+
+- A path to a file within the project root (the directory that contains the
+  default Cypress configuration file). Eg: `'path/to/file.json'`
+- `@alias` - An alias of any type, previously stored using `.as()`. Eg:
+  `'@alias'`
+- A `Buffer()` containing binary data, such as that returned by
+  `cy.readFile('file.json', { encoding: null })`. Eg: `Buffer.from('foo')`
+- An object with a non-null `contents` property, specifying details about the
+  file. Eg: `{contents: '@alias', fileName: 'file.json'}`
+
+If an object is provided, it can have the following properties.
+
+| Option         | Description                                                                                                                                                                                                                                        |
+| -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `contents`     | The contents of the file. This can be a string shorthand as described above, a `Buffer()` containing binary data or a non-Buffer object, which will be converted into a string with `JSON.stringify()` and `utf8` encoded.                         |
+| `fileName`     | The name of the file. If `contents` is a path on disk, this defaults to the actual filename. In any other case, this defaults to an empty string.                                                                                                  |
+| `lastModified` | The file's last modified timestamp, in milliseconds elapsed since the UNIX epoch (eg. [`Date.prototype.getTime()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime)). This defaults to `Date.now()`. |
+
+**<Icon name="angle-right"></Icon> options** **_(Object)_**
+
+Pass in an options object to change the default behavior of `.selectFile()`.
+
+| Option                       | Default                                                                        | Description                                                                                                                                       |
+| ---------------------------- | ------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `action`                     | `'select'`                                                                     | Switches selectFile between modes. Valid values are `select` and `drag-drop`. See [Actions](#Actions) below for more details.                     |
```

Comment:
Can you either format selectFile as `.selectFile()` here or remove "selectFile between"?

Reply:
Removed extra words.